docs/_templates/layout.html

```diff
@@ -20,25 +24,30 @@
   {%- endblock -%}
 
   {#- CSS #}
-  <link rel="stylesheet" href="{{ fixpath(pathto('_static/' + style, 1)) }}" type="text/css" />
-  <link rel="stylesheet" href="{{ fixpath(pathto('_static/pygments.css', 1)) }}" type="text/css" />
-  <link rel="stylesheet" href="{{ fixpath(pathto('_static/custom.css', 1)) }}" type="text/css" />
+  {%- if sphinx_version_info < (4, 0) -%}
+    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
+  {%- endif %}
   {%- for css in css_files %}
     {%- if css|attr("rel") %}
-      <link rel="{{ css.rel }}" href="{{ fixpath(pathto(css.filename, 1)) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
+      <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
     {%- else %}
-      <link rel="stylesheet" href="{{ fixpath(pathto(css, 1)) }}" type="text/css" />
+      <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
     {%- endif %}
   {%- endfor %}
 
   {%- for cssfile in extra_css_files %}
-    <link rel="stylesheet" href="{{ fixpath(pathto(cssfile, 1)) }}" type="text/css" />
```

Comment:
at least i am not getting some 404 but you can run the build command and try it.

Reply:
added it again but not sure if it is really necessary

Reply:
It might be needed for older versions. Perhaps we ought to add a comment below `def fixpath()` explaining that it only exists for legacy reasons and should not be copied into new projects.